main/security.bs

```diff
@@ -3,6 +3,33 @@ Security Considerations {#security}
 
 Issue: Write Security Considerations section.
 
+Solid data pods [disable all cross-origin protections](#cors-server) in browsers
+because resource access is governed explicitly by [Web Access Control](#wac).
+As such,
+data pods MUST NOT rely on cross-origin protection
+for shielding access to resources.
+While this ensures that unauthorized resource access will not occur,
+additional security measures MAY be needed
+to prevent metadata in error responses from leaking.
+For instance,
+a malicious app could probe multiple servers
+to check whether the response status code is `401` or `403`,
+or could try to access an error page
+from an intranet server within the user agent's private network
+to extract company names or other data.
+To mitigate this,
+when a request from an untrusted the `Origin` arrives,
+the data pod MAY set the status code of error responses to `404`
+and/or anonymize or censor their contents.
+
+A data pod MUST NOT assume that the user agent is a regular Web browser,
```

Comment:
Perhaps it doesn't necessarily need to be mentioned in the document, but for now, can you clarify (in the comments) why a pod might make a decision based on the requesting agent? By default, it is preferable to not assume anything, so is this MUST NOT here to just to prevent any potential issues?

Reply:
It is for security reasons. In some discussions in the past, people were assuming that the presence of an `Origin` header meant that the client had the security model of a browser. Will try to describe that.